tests/main/desktop-portal-screenshot/task.yaml

```diff
@@ -57,7 +57,7 @@ execute: |
     # Taking a screenshot is not supported in kinetic and lunar yet.
     # The error displayed is the following:
     # dbus.exceptions.DBusException: org.freedesktop.DBus.Error.UnknownMethod: No such interface \u201corg.freedesktop.portal.Screenshot\u201d on object at path /org/freedesktop/portal/desktop
-    if os.query is-ubuntu-ge 23.04; then
+    if os.query is-ubuntu-ge 23.10; then
```

Comment:
"kinetic and lunar yet" probably should be changed to just "mantic" now.

Reply:
updated. Thanks for the review